suzieq/poller/worker/nodes/node.py

```diff
@@ -1470,90 +1471,21 @@ async def init_node(self):
                         "data": output}]
                     await self._parse_boottime_hostname(res, None)
             self._session = aiohttp.ClientSession(
-                conn_timeout=self.connect_timeout,
-                connector=aiohttp.TCPConnector(ssl=False),
-            )
+                    conn_timeout=self.connect_timeout,
+                    connector=aiohttp.TCPConnector(ssl=False),
```

Comment:
Is SSL always False or is this the way you specify to ignore certificate verification?

Reply:
It is to ignore certificate validation, [reference](https://docs.aiohttp.org/en/stable/client_reference.html#:~:text=SSL%20validation%20mode).